Source/Examples/WPF/ExampleBrowser/MainWindowViewModel.cs

```diff
@@ -100,7 +100,7 @@ private void CoerceRenderer()
             this.RaisePropertyChanged(nameof(this.CanvasModel));
         }
 
-        public IEnumerable<Renderer> Renderers => Enum.GetValues(typeof(Renderer)).Cast<Renderer>();
+        public static IEnumerable<Renderer> Renderers => Enum.GetValues(typeof(Renderer)).Cast<Renderer>();
```

Comment:
This change breaks the WPF ExampleBrowser - the property is used by a data binding, which does not work if the property is static. That is an example of why I think that making things `static` should be considered case by case, and not done with an automatic solution-wide refactoring.

Reply:
That's a fair argument. I'm removing that one now

Reply:
And yeah, automated refactoring could conflict with tools that rely on reflection, so I understand what you mean